app/assets/stylesheets/themes/_light-theme.scss

```diff
@@ -56,6 +56,12 @@ $pulsate-red-end: #e87f84;
 
 $modal-bg-color: #dadada;
 
+$new-positive-color: #22d173;
```

Comment:
Please use meaningful variable names here as well. If it's an alternative to the commonly used positive color, call it `positive-color-alternative`

Reply:
Will do, thanks.